packages/webdriverio/webdriverio.d.ts

```diff
@@ -12,6 +12,8 @@ type ElementPromise = Omit<WebdriverIO.Element,
     | 'elementId'
     | 'element-6066-11e4-a52e-4f735466cecf'
     | 'ELEMENT'
+    | 'dragAndDrop'
```

Comment:
Why do we need to define commands here? They have to be declared once in webdriverio.core.d.ts and automatically wrapped with Promise here

Reply:
Here the problem is with arguments not return types. Element in dragAndDrop in webdriverio-core.d.ts refers to WebdriverIO.Element always but in async mode it needs to refer to WebdriverIOAsync.Element